docs/configuration/install.md

```diff
@@ -502,93 +541,223 @@ ExecStop=/usr/bin/java $JAVA_OPTS -jar $ILSERVER_JAR $ILSERVER_INI stop
+using ```ssmtp```:
+
+On Debian/Ubuntu execute:
+```
+apt-get install ssmtp
+```
+
+On RHEL/CentOS execute:
+```
+yum install ssmtp
+```
+
+The configuration file for SSMTP (e.g. ```/etc/ssmtp/ssmtp.conf ```) may look as
+follows:
 
-Due the Call for Bids "Documentation for  'Secure ILIAS'", the content is moved to an own file.
+```
+#
+# Config file for sSMTP sendmail
+#
+# The person who gets all mail for userids < 1000
+# Make this empty to disable rewriting.
+root=yourmail@mail.com
 
-See [Hardening and Security Guidance](secure.md)
+# The place where the mail goes. The actual machine name is required no
+# MX records are consulted. Commonly mailhosts are named mail.domain.com
+mailhub=smtp.yourmail.com
 
-<a name="customizing-ilias"></a>
-# Customizing ILIAS
+# Where will the mail seem to come from?
+rewriteDomain=yourservername.org
 
-If you need to customize your ILIAS installation you MUST NOT edit the core files, otherwise you will not be able to update your installation in a timely manner (e.g. due to security fixes).
+# The full hostname
+hostname=yourserver.example.com
 
-You can find proper ways to customize ILIAS in the [ILIAS Development Guide](http://www.ilias.de/docu/goto_docu_pg_29964_42.html):
+# Are users allowed to set their own From: address?
+# YES - Allow the user to specify their own From: address
+# NO - Use the system generated From: address
+FromLineOverride=YES
+```
+
+<a name="webdav-configuration"></a>
+## Configure WebDAV (optional)
 
-  * [Plugins and Plugin Slots - ILIAS Development Guide](http://www.ilias.de/docu/goto.php?target=st_27029)
-  * [Custom Styles](/templates/Readme.md#custom-styles)
+The recommended webserver configuration is either **Apache with mod_php** or
+**Nginx with PHP-FPM (> 1.3.8)**. Do NOT use **Apache with PHP-FPM** if you
+use WebDAV.
 
-<a name="plugin-repository"></a>
-## Plugin Repository
+<a name="webdav-windows-explorer"></a>
+### WebDAV with Windows Explorer
+Because of a special behaviour in the Windows Explorer, it sometimes fails to
+add a WebDAV connection with the error code `0x80070043 The Network Name
+Cannot Be Found`.
+
+To prevent this behaviour, add the following rewrite rules to a .htaccess file
+in your webroot or to the corresponding section of the configuration of your
+webserver:
+
+```
+RewriteCond %{HTTP_USER_AGENT} ^(DavClnt)$
+RewriteCond %{REQUEST_METHOD} ^(OPTIONS)$
+RewriteRule .* "-" [R=401,L]
+```
+
+<a name="webdav-mac-finder"></a>
+### WebDAV with Mac Finder
+To upload files, the WebDAV Client *Finder* on Mac uses chunked transfer encoding.
+Some webservers can't handle this way of uploading files and are serving ILIAS
+an empty files, which results in an empty file object on ILIAS. Due to a bug in
+apache, the configuration of **Apache with PHP-FPM** does not work with the *Mac
+Finder*. If you use WebDAV on your ILIAS installation, we recommend to either
+use **Apache with mod_php** or **Nginx with PHP-FPM (> 1.3.8)**.
```

Comment:
```suggestion
use **Apache with mod_php** or **Nginx with PHP-FPM (> 1.3.8)**. Please find more information in the corresponding [README](../../Services/WebDAV/README.md)
```
...and then we get rid of the rest of the information in here concerning WebDAV. We might want to do this for all optional elements to keep this file short and hand responsibility over to maintainers. What do you think?

Reply:
Something like this would be great.